resources/assets/js/projects/title.vue

```diff
@@ -61,11 +61,10 @@ export default{
             setTimeout(() => location.href = this.redirectUrl, 2000);
         },
         deleteProject() {
-            let confirmed = confirm(`Do you really want to delete the project ${this.project.name}?`);
-
-            if (confirmed) {
+            let inputproject = prompt(`Are you scure you want to delete "${this.project.name}"? Please type the project name to confirm.`);
+            if (inputproject == this.project.name) {
```

Comment:
```suggestion
            if (inputproject === this.project.name) {
```
The identity operator `===` also checks the arguments types, so it's a bit safer to use.